conda/dev-environment-linux.yml

```diff
@@ -46,4 +47,5 @@ dependencies:
  - twine
  - unzip
  - wheel
+ - websocketpp
```

Comment:
It's going to be some time before there is a `csp-extensions` repo in any form, and certainly for C++ extensions where there may be ABI boundaries. I don't think we'd want to block this PR on waiting (since there are active uses of a websocket client).